src/AprsISLibrary/AprsIsConnection.cs

```diff
@@ -20,6 +20,22 @@
     /// </summary>
     public class AprsIsConnection
     {
+        private readonly ITcpConnection tcpConnection;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AprsIsConnection"/> class.
+        /// </summary>
+        /// <param name="tcpConnection">A TcpConnection to use for communication.</param>
```

Comment:
Found another correction here. This should really say it's an `ITcpConnection` rather than just `TcpConnection` we as are allowing the abstraction of the interface here.

Also, we can reference types in the XML comments here, which is pretty cool. It allows the editor (such as VS Code) to show more detailed information in the auto completion tooltips. See above where I used the `<see cref="AprsIsConnection"/>` element, that will show up a link to the `AprsIsConnection` documentation. More importantly, this will cause the compiler to check if the type exists. If it doesn't, an error will be raised. This ensures that if a type name changes, we cannot forget to update it in the comments about it. This is a similar motivation to the use of `nameof` which I mentioned in another comment.

```suggestion
        /// <param name="tcpConnection">An <see cref="ITcpConnection"/> to use for communication.</param>
```